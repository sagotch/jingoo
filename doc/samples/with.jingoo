{% with foo = 10, hoge = 20 -%}
<dl>
  <dt>foo</dt><dd>{{ foo }}</dd>
  <dt>hoge</dt><dd>{{ hoge }}</dd>
</dl>
{%- endwith %}
