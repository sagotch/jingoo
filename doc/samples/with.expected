<dl>
  <dt>foo</dt><dd>10</dd>
  <dt>hoge</dt><dd>20</dd>
</dl>
